arch/x86/Kconfig

```
@@ -479,6 +479,14 @@ config X86_KPTI
 	  user thread interrupts and system calls, and significant footprint
 	  increase for additional page tables and trampoline stacks.
 
+config X86_EFI
+	bool "Enable EFI support"
```

As per the guidelines, please remove `Enable` and `support`:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe just write:
```suggestion
	bool "EFI"
```

---

Ok, did not know about such rule. Sounds a bit blunt in a menu to just read "EFI" but ok

---

> Sounds a bit blunt in a menu to just read "EFI" but ok

You are always allowed to rephrase to a more descriptive title.
Just avoid the `Enable` and `support` as this adds no informational value.

Perhaps a title like:
```
	bool "EFI boot"
```
Already now, you'll notice that `boot` adds informational value to users just looking at the menu, as `boot` ensures that users couple `EFI` word to correct context (generally EFI is foreign in embedded systems).

There are many ways a title can be improved and provide extra informational value when removing the words `enable` and `support`. Better use the space for words of informational value, than to use those two words :wink: